presentation/src/main/java/team/jsv/icec/ui/main/SelectMosaicEditFragment.kt

```diff
@@ -15,29 +13,20 @@ class SelectMosaicEditFragment :
 
     private val viewModel: MosaicViewModel by activityViewModels()
 
-    override fun initView() {
-        setButtonStroke()
-    }
-
     override fun onResume() {
         super.onResume()
 
+        initView()
+    }
+
+    override fun initView() {
         binding.buttonMosaic.setOnClickListener {
             navController.navigate(R.id.action_selectMosaicEditFragment_to_faceSelectFragment)
-            viewModel.setScreen(ScreenStep.SelectFace)
-            viewModel.originalImage.value?.let { originalImage -> viewModel.getFaceList(originalImage) }
+            viewModel.originalImage.value?.let { it1 -> viewModel.getFaceList(it1) }
```

Comment:
it1 대신에 it을 사용하거나, image -> 요런걸로 사용해주는건 어떨가요:?
```suggestion
            viewModel.originalImage.value?.let { image -> viewModel.getFaceList(image) }
```

Reply:
넵 변경했습니다!